jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/jdbc/JdbcMetadataReader.java

```diff
@@ -252,7 +252,7 @@ private static String findSchema(final String schema, final DatabaseMetaData dbM
         final List<SqlDialect.MappedTable> tablesMapped = new ArrayList<>();
         // List<String> tableComments = new ArrayList<>();
```

Comment:
Remove all commented code.

Reply:
leave that to an IDE in a refactoring branch